doc/classes/ItemList.xml

```diff
@@ -179,6 +179,18 @@
 				[b]Warning:[/b] This is a required internal node, removing and freeing it may cause a crash. If you wish to hide it or any of its children, use their [member CanvasItem.visible] property.
 			</description>
 		</method>
+		<method name="insert_item">
+			<return type="void" />
+			<param index="0" name="idx" type="int" />
+			<param index="1" name="text" type="String" />
+			<param index="2" name="icon" type="Texture2D" default="null" />
+			<param index="3" name="selectable" type="bool" default="true" />
+			<description>
+				Inserts an item at the specified index in the item list with the specified text.
```

Comment:
```suggestion
				Inserts an item with the specified text at the specified index in the item list.
```
Otherwise it seems like the item list has the specified text